verify_filenames.py

```diff
@@ -63,21 +63,28 @@
 
     for f in folder.iterdir():
 
-       # Test if there are any nested folders
+        # Test if there are any nested folders
```

Comment:
Not sure why whitespace needs changing here?

Reply:
that particular block was indented with 3 spaces instead of 4